src/components/txNotification/transactions.ts

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
Why this file not in `stores` folder?

Reply:
I thought to make this component+store reusable and ready to drop in other projects. What do you think? Is it a good idea? What are the best practices for this?

Reply:
also that's why I created a separate type definition `DetailedInflightTxReceipt` so this "component" doesn't depend on types used in host project (`InflightTxReceipt`).

Yes, these names suck. I tried to make the whole thing simpler to understand and to use, but this "promise of promise of tx receipt" mess still gives me some headache

Reply:
Ok, makes sense